test/unit/task/dump_tests.rb

```diff
@@ -31,7 +31,12 @@ class InitTests < UnitTests
       now = Factory.time
       Assert.stub(Time, :now){ now }
 
-      set_dk_dumpdb_script_param
+      @dump_cmds = dump_cmds = Factory.integer(3).times.map{ Factory.string}
```

Comment:
@kellyredding - really minor, I think you are missing a space after `Factory.string`